docs/content/deployment/guides/aws.mdx

```diff
@@ -50,7 +50,7 @@ The Deploying on ECS example on GitHub demonstrates how to configure the [Docker
 - A Dagit container for loading and launching jobs
 - A `dagster-daemon` container for managing a run queue and submitting runs from schedules and sensors
 - A Postgres container for persistent storage
-- A container with user job code
+- A code server container that provides information about your code to Dagit and the daemon.
```

Comment:
```suggestion
- A code server container that provides information about your code to the Dagster UI and the daemon
```

A few things:
- Dagit should be Dagster UI
- Nitpicky, but removing the period for consistency